packages/visualizations/addon/templates/components/cell-renderers/dimension.hbs

```diff
@@ -1,6 +1,10 @@
 {{!-- Copyright 2017, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
 {{#with (get data (concat column.field.dimension '|desc' )) as |description|}}
-    <span title={{description}}>{{description}}</span>
+    {{#with (get data (concat column.field.dimension '|id' )) as |id|}}
```

Comment:
This is done on line #9 as well. Can we make this the outer `with`?

Reply:
would that actually fix conditions for all 4 states?

Reply:
gonna refactor this so that the component makes this decision rather than the template.